cms/envs/common.py

```diff
@@ -1208,3 +1208,7 @@
 # to generating test databases will discover and try to create all tables
 # and this setting needs to be present
 OAUTH2_PROVIDER_APPLICATION_MODEL = 'oauth2_provider.Application'
+
+# Used with Email sending
+RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS = 5
```

Comment:
optional suggestion: import these from lms.envs.common instead of redefining them